### DISCUSSION TITLE
Sweeping a State Variable Using PF

### DISCUSSION THREAD

**Discussion by jsevic** [idaholab/moose#26049] [Q&A Modules: Phase field]
This question may be for more general MOOSE use, at the moment I have a PF simulation and one of the variables is an applied external electric potential V. Usually I run a transient simulation for a fixed V, however I would like to simulate to steady-state for say V = 0 to 1 in steps in 0.1 V.

Thank you.

**Comment by GiudGiud** [comment]
Hello

Can you use a FunctionAux?
Is the variable a nonlienar variable or an auxiliary variable?

Guillaume
